cachi2/core/package_managers/rpm/redhat.py

```diff
@@ -17,12 +17,19 @@ class LockfilePackage(BaseModel):
     size: Optional[int] = None
 
 
+class LockfileModuleMetadata(LockfilePackage):
+    """Module metadata item; represents module metadata file."""
+
+    repoid: str
+
```

Comment:
why aren't url, size, and checksum mentioned here? Especially `url` since it's a mandatory field without which we can't download anything. What about the rest, are those optional just like for a regular RPM?

Reply:
Those fields are not mentioned because the only difference from the regular PRM is that a repoid is required.

Reply:
But if I omit a URL in the lockfile, how is cachi2 supposed to download the metadata? The URL field must be required as well just like for a package.

Reply:
As it's already 'required' in the parent class, cachi2 will raise error.

Error: PackageManagerError: RPM lockfile 'rpms.lock.yaml' format is not valid: '('arches', 0, 'module_metadata', 0, 'url'): Field required'
  Check the correct format and whether any keys are missing in the lockfile.

Reply:
Darn, I overlooked the inherited model, all good.